web/src/lib/components/shared-components/search-bar/search-people-section.svelte

```diff
@@ -38,15 +40,24 @@
     }
     selectedPeople = selectedPeople;
   }
+
+  const filterPeople = (list: PersonResponseDto[], name: string) => {
+    const nameLower = name.toLowerCase();
+    return name === '' ? list : list.filter((p) => p.name.toLowerCase().startsWith(nameLower));
```

Comment:
You don't need `name === ''` as `''` already is falsy.
```suggestion
    return name ? list : list.filter((p) => p.name.toLowerCase().startsWith(nameLower));
```

Reply:
Also, do we want to use `includes` instead for a more fuzzy-ish search?

Reply:
I actually originally had include but noticed the filter on people page does starts with. Starts with probably makes sense for names unless you want to search for all people with last name

Reply:
I'll play with the if again tonight. I thought I tried that first and it didn't work. But it was late last night